### PR TITLE
WIP: Expand on file-structure page.

### PR DESCRIPTION
**Pull request by Centrus007**

### CODE REVIEW

File: docs/developer/documentation.rst [hunk 1]
```diff
@@ -29,9 +29,9 @@ Within the root pymedphys directory run the following command:
 
 .. code:: bash
 
-    sphinx-autobuild docs docs/_build/html
+    sphinx-autobuild -p 7070 docs docs/_build/html
 
-Then within a web browser go to http://127.0.0.1:8000
+Then within a web browser go to http://127.0.0.1:7070
```
Comment (SimonBiggs) [inline docs/developer/documentation.rst]: Weirdly enough, `Mosaiq` hosts a webserver on port 8000. Given quite a few of our documentation editors might also be Mosaiq users, I figure it is best to dodge the default port of `sphinx-autobuild`

File: docs/developer/contributing.rst [hunk 2]
```diff
@@ -164,29 +164,30 @@ This will download all of PyMedPhys to your local machine.
...
 
 .. WARNING::
 
-    In the event that you uninstall Python, it is possible that `nbstripout`
+    In the event that you uninstall Python, it is possible that ``nbstripout``
     ends up disabled. Stay prudent, and be extra cautious when working with
     sensitive information stored within a notebook in a Git repository.
 
-To install `nbstripout` run the following within the pymedphys directory:
+To install ``nbstripout`` run the following within the pymedphys directory:
```
Comment (SimonBiggs) [inline docs/developer/contributing.rst]: Fixed these while I saw them. To do inline code blocks in reST one can surround with double "`"

Comment (Centrus007) [reply docs/developer/contributing.rst]: Good tip! Cheers :)

File: docs/developer/documentation.rst (hunk 1 above)
Comment (Centrus007) [reply docs/developer/documentation.rst]: No worries :)

**Pull request merged**